package.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "eclint": "^1.1.5",
+    "istanbul": "^0.4.5",
```

Comment:
This was added by running `make cover` on local. We should check this missing dependency in

https://github.com/gotwarlost/istanbul/releases/tag/v0.4.5